modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/PredefinedValuesTable.tsx

```diff
@@ -228,32 +229,10 @@ export default function PredefinedValuesTable({
 			return objectFields;
 		};
 
-		const deletePredefinedValueField = ({itemData}: {itemData: Item}) => {
-			const {name} = itemData;
-
-			if (objectFieldsMap.get(name)?.required) {
-				openToast({
-					message: Liferay.Language.get(
-						'required-fields-cannot-be-deleted'
-					),
-					type: 'danger',
-				});
-
-				return;
-			}
-
-			const newPredefinedValues = predefinedValues?.filter(
-				(field) => field.name !== name
-			);
-
-			setValues({
-				parameters: {
-					...values.parameters,
-					predefinedValues: newPredefinedValues,
-				},
-			});
-		};
+		return getSelectedFields;
+	}, [predefinedValues, objectFieldsMap]);
 
+	const handleAddFields = useMemo(() => {
```

Comment:
Same as before: This returns a function that returns a function. It looks like a use case for `useCallback`. You could return the nested function directly. Check [React docs](https://react.dev/reference/react/useMemo#memoizing-a-function).